.github/workflows/tests.yaml

```diff
@@ -10,8 +10,9 @@ jobs:
   test:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
-        php-versions: ['5.6', '7.3', '7.4', '8.0']
+        php-versions: ['5.6', '7.3', '7.4', '8.0', '8.2']
```

Comment:
Not for this, but I'd really like to discuss removing support for 5.6, it's been EOL for over 5 years now https://www.php.net/eol.php

Reply:
I think Our Parks are still using it... need to confirm that though